packages/block-library/src/comment-template/index.php

```diff
@@ -5,6 +5,84 @@
  * @package WordPress
  */
 
+/**
+ * Convert the flat array of comments to a nested array
+ * containing comments with a `comment_children` property.
+ *
+ * @param WP_Comment[] $comments  - Array of comments.
+ * @return array
+ */
+function convert_to_tree( $comments ) {
+	$table = array();
+
+	// If there are no comments, we can return early an empty array.
+	if ( ! isset( $comments ) || empty( $comments ) ) {
```

Comment:
`empty` is a language construct that also performs `isset` check:

```suggestion
	if ( empty( $comments ) ) {
```

Reply:
TIL 😃